### ISSUE TITLE
Feature brick creates all the folders on the way to the selected folder 

### ISSUE THREAD

**Issue by nivisi** [monstar-lab-oss/flutter-bricks#1]
Just tried right-clicking on a folder and creating a feature brick. Here's what I've got:

<img width="520" alt="Screenshot 2024-05-20 at 2 00 52 PM" src="https://github.com/ml-opensource/flutter-bricks/assets/33932162/b3bd8fdc-d5b5-4d44-8cf7-7ee7eea1ed3c">

### Steps to reproduce

1. Install ML bricks
2. Right click on a folder
3. Select the `feature` brick and finish making it
4. Observe the result

**Comment by utpal-barman** [comment]
wait, how do you "right click on feature and select the feature brick"? are you using any plugin?

Our mason bricks are supposed to make from the command line, assuming we are always in the root folder

**Comment by nivisi** [comment]
@utpal-barman yep, there's a mason extension for VS Code. So I created a "subfeature" within a feature folder, but got all the `lib/presentation/...` folders
